src/bin/agent/CommandLockAll.ts

```diff
@@ -16,9 +17,11 @@ class CommandLockAll extends CommandPolykey {
     this.addOption(binOptions.clientPort);
     this.action(async (options) => {
       const { default: PolykeyClient } = await import('../../PolykeyClient');
+      const { default: WebSocketClient } = await import(
+        '../../websockets/WebSocketClient'
+      );
       const { default: Session } = await import('../../sessions/Session');
-      const utilsPB = await import('../../proto/js/polykey/v1/utils/utils_pb');
-
+      const { clientManifest } = await import('../../client/handlers');
```

Comment:
This imports EVERYTHING! The `client/handlers/index.ts` is importing the entire project.

Don't do this. Remember the whole reason we are doing this is `await import` is to specialise to the only the code necessary for the command.

You're making every command dynamically load the entire project source with `await import('../../client/handlers');`

You need to have a more specific import.

The original thing was just `utilsPB` which was just a simple structure for utilities.

Now this ends up slowing the command execution.

I see the way you're creating the `clientManifest`. I think it's too heavy, either we need a different way, or can we do without having a manifest, or can we only import **exactly** what we need without going through the `index.ts` imports.

Reply:
The core of this is we need a way to define the manifest without importing everything. The reason everything is being imported because the client caller definitions are defined in the same file of the handler.  So we're implicitly importing the handler dependencies.

The only thing the callers need are the types involved. but we need to keep them in a logical place close to the handlers because they need to be created and updated side by side. The reason for this is that the client and agent definitions are closely coupled by type, not enforced by any type system and if they don't match exactly we end up with type bugs that are hard to debug without intimate knowledge of the system. 

As for a fix for this. We need to define the server and client manifest in seperate files and we can't defined the callers in the handlers file. I need to make a `clientManifest.ts` file and possibly a server variant for this. I'm not sure if these files should be kept in the `handlers` directory or the `client` directory. 

I'll keep them in the handlers directory for now.

Reply:
The `clientManifest` and `serverManifest` are now in their own respective files, `clientManifest.ts` and `serverManifest.ts`. They are both still exported by the index. 

We are now importing just the `clientManifest` for use with the `rpcClient`. This should only depend on type imports now. 

To get any more specific than this then we'd need separate files for each caller definition. That would be too much to maintain for little benifit.